app/controllers/editions_controller.rb

```diff
@@ -27,6 +30,25 @@ def show
   alias_method :metadata, :show
   alias_method :unpublish, :show
 
+  def update
+    params = permitted_params
```

Comment:
I'd be wary of calling this variable `params` as that is already the name of a field accessible in rails controllers (maybe use `update_params` instead).

Reply:
I agree, I think it will cause confusion. I have added a comment about the permitted_params method below and I think if the .permit was changed to the pattern I have mentioned, then you could maybe change the line below to:

`@resource.assign_attributes(permitted_params)`